package.json

```diff
@@ -28,14 +28,14 @@
     "@uirouter/core": "^6.1.0"
   },
   "devDependencies": {
-    "@angular/animations": "^17.2.1",
-    "@angular/common": "^17.2.1",
-    "@angular/compiler": "^17.2.1",
-    "@angular/compiler-cli": "^17.0.0",
-    "@angular/core": "^17.2.1",
-    "@angular/platform-browser": "^17.2.1",
-    "@angular/platform-browser-dynamic": "^17.2.1",
-    "@angular/upgrade": "^17.2.1",
+    "@angular/animations": "^18.0.6",
+    "@angular/common": "^18.0.6",
+    "@angular/compiler": "^18.0.6",
+    "@angular/compiler-cli": "^18.0.6",
+    "@angular/core": "^18.0.6",
+    "@angular/platform-browser": "^18.0.6",
+    "@angular/platform-browser-dynamic": "^18.0.6",
+    "@angular/upgrade": "^18.0.6",
     "@types/angular": "^1.8.9",
     "@types/jquery": "^3.5.6",
     "@uirouter/angular": "^13.0.0",
```

Comment:
This needs to be 14

Reply:
Updated

Reply:
I think the examples folder needs updating to angular 18, etc,

Reply:
Right, I missed that one. I updated it now.